testCommon/src/test/resources/common.sh

```diff
@@ -36,10 +36,12 @@ function create_repo() {
   cd $dir
   shift
   git init $@
-  mkdir -p .git/hooks/
-  local hook_path=.git/hooks/machete-status-branch
-  echo "$status_branch_hook" > $hook_path
-  chmod +x $hook_path
+  if [ "$(expr substr $(uname -s) 1 10)" != "MINGW64_NT" && "$(expr substr $(uname -s) 1 10)" != "MINGW32_NT" ]; then
```

Comment:
Pls open an issue on making this work on Windows (sth like `Enable running machete-status-branch hook in tests on Windows`) + reference it in a TODO here